### ISSUE TITLE
GO:0022857 | transmembrane transporter activity | IBA with PTN000248895 ,

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5533]
* **PTHR ID & PTN node:**

GO:0022857 | transmembrane transporter activity | IBA with PTN000248895 , MGI:2151054 , MGI:2151060

* **Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPCC4B3.03c mam310

The transporter annotations come from mouse, but it doesn't look like a transporter structurally

Also, it does not appear to be a transporter in mouse according to the descriptions
Cnnm2 https://www.informatics.jax.org/marker/MGI:2151054 (described as cyclin M2)
Cnnm4 https://www.informatics.jax.org/marker/MGI:2151060 (described as cyclin M4)
(although it doesn't look like a cyclin either)

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

Erroneous source?
@LiNiMGI  could you take a look?
Thanks
Val

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by marcfeuermann** [comment]
There are several publications that show that these proteins are magnesium transporters, not only in mouse (PMID:24706765, 21397062, 24339795), but also in S. cerevisiae (PMID: 35087208), C. elegans (PMID: 27564576) or Drodophila (PMID: 33242000).
So, I would keep this annotation for the moment, unless we get good evidence for the contrary.

**Comment by ValWood** [comment]
OK, I was interested in seeing more support since the mouse proteins are described as "cyclins", and the SGD  MAM3 isn't annotated as a transporter, but maybe it could be? @suzialeksander ?
At present mouse seems to be the only annotation, so I ended up annotating the product as "multi-pass membrane and CBS domain protein, magnesium extrusion factor" because I wasn't sure we had the evidence that it was directly transporting magnesium.

**Comment by ValWood** [comment]
Actually, I see Drosophila has the annotation already. OK ignore.
